roqoqo/README_roqoqo.md

```diff
@@ -9,21 +15,46 @@ What roqoqo is:
 * A toolkit to represent quantum programs including circuits and measurement information
 * A thin runtime to run quantum measurements
 * A way to serialize quantum circuits and measurement information
-* A set of optional interfaces to devices, simulators and toolkits (e.g. [qoqo_quest](https://github.com/HQSquantumsimulations/qoqo-quest), [qoqo_mock](https://github.com/HQSquantumsimulations/qoqo_mock), [qoqo_qasm](https://github.com/HQSquantumsimulations/qoqo_qasm))
+* A set of optional interfaces to devices, simulators and toolkits (e.g. [qoqo_quest](https://github.com/HQSquantumsimulations/qoqo-quest), [qoqo_qiskit](https://github.com/HQSquantumsimulations/qoqo-qiskit), [qoqo_for_braket](https://github.com/HQSquantumsimulations/qoqo-for-braket), [qoqo_iqm](https://github.com/HQSquantumsimulations/qoqo_iqm))
 
 What roqoqo is **not**:
 
 * A decomposer translating circuits to a specific set of gates
 * A quantum circuit optimizer
 * A collection of quantum algorithms
 
-## roqoqo
 
-[![Crates.io](https://img.shields.io/crates/v/roqoqo)](https://crates.io/crates/roqoqo)
-[![GitHub Workflow Status](https://github.com/HQSquantumsimulations/qoqo/workflows/ci_tests/badge.svg)](https://github.com/HQSquantumsimulations/qoqo/actions)
-[![docs.rs](https://img.shields.io/docsrs/roqoqo)](https://docs.rs/roqoqo/)
-![Crates.io](https://img.shields.io/crates/l/roqoqo)
-[![codecov](https://codecov.io/gh/HQSquantumsimulations/qoqo/branch/main/graph/badge.svg?token=S1IN066V2W)](https://codecov.io/gh/HQSquantumsimulations/qoqo)
+If you are looking for a comprehensive package that integrates all these features, we invite you to explore our [HQStage](https://cloud.quantumsimulations.de/) software.
+
+
+## Installation
+
+To use roqoqo in a Rust project simply add
+
+```TOML
+roqoqo = {version="1.0"}
+```
+
+to the `[dependencies]` section of the project Cargo.toml.
+
+
+When using qoqo in a rust project providing a python interface add
+
+```TOML
+qoqo = {version="1.0", default-features=false}
+```
+
+to the `[dependencies]` section of the project Cargo.toml.
+
+A source distribution now exists but requires a Rust install with a rust version > 1.47 and a maturin version { >= 0.12, <0.13 } in order to be built.
```

Comment:
This is out of date, the rust version should be 1.76, and the maturin one should be >=1.0, <2.0

Reply:
But just >=1.0 should be enough for maturin

Reply:
ok, I'll update it.